verification/curator-service/auth.md

```diff
@@ -12,19 +12,18 @@ The different roles are:
 
 - admin
 - curator
-- reader
 
 A user can have no roles.
 
 Here are the permissions granted to each roles:
 
 - Admin
   - List users based on their roles
-  - Assign/unassign role to users 
+  - Assign/unassign role to users
 - Curator
   - CRUD cases and ingestion sources
-- Reader
-  - Read-only access to cases and sources
+
+By default all logged-in users have read-only access to the linelist data and sources used to populate it.
```

Comment:
Readers don't have access to sources

Reply:
they do, on purpose so that folks can see where the data came from (it was the case before, this didn;t change with this PR).